app/build.gradle

```diff
@@ -14,11 +14,10 @@ android {
 
     defaultConfig {
         applicationId "io.lbry.browser"
-        minSdkVersion 21
-        targetSdkVersion 29
+        minSdkVersion 24
+        targetSdkVersion 30
```

Comment:
These need to be left as they are. `targetSdkVersion` should only be changed if absolutely necessary (required by Google Play). `minSdkVersion` indicates the minimum supported Android version (which is 5.0, and we don't intend to stop supporting that yet).